packages/terra-consumer-nav/src/NavPropShapes.js

```diff
@@ -6,14 +6,18 @@ const navItemShape = {
    */
   text: PropTypes.string.isRequired,
   /**
-   * The path the profile link would redirect to.
+   * The path the link would redirect to.
    */
-  uri: PropTypes.string.isRequired,
+  url: PropTypes.string,
```

Comment:
There may be a case where the url is not needed. Like clicking a navigation item expand a sub item like the following.
![screen shot 2017-08-24 at 3 20 59 pm](https://user-images.githubusercontent.com/21693381/29684611-9a7b96c4-88e0-11e7-8994-3f5020d7f0c6.png)